esp32/lte/lteppp.c

```diff
@@ -654,22 +658,30 @@ static bool lteppp_send_at_cmd_exp (const char *cmd, uint32_t timeout, const cha
         }
 #endif
         // flush the rx buffer first
-        uart_flush(LTE_UART_ID);
+        if(!expect_continuation || 
+            (len >= 2 && cmd[0] == 'A' && cmd[1] == 'T')) // starts with AT
+            uart_flush(LTE_UART_ID);
```

Comment:
{curly braces} please. there's another single line if just below that also has them, and with the two line conditional